src/tags/object/Text.js

```diff
@@ -85,6 +85,10 @@ const Model = types
       self._ref = ref;
     },
 
+    needsUpdate() {
+      self._update = self._update + 1;
+    },
```

Comment:
thanks! I was about to ask you about this :)

Reply:
yeah, i noticed this one when was removing excess logs :) it was supposed to be removed in `RichText` but i missed the file, probably